### PR TITLE
feat(RSVP): Prevent classic rsvp form showing on blocks posts.

### PR DESCRIPTION
**Pull request by Camwyn**
Add a check to RSVP.php to test if the user has enabled the MT blocks editor, and the post has blocks. If so, bail before adding the classic front-end RSVP form.

:ticket: https://central.tri.be/issues/124394

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -138,6 +138,7 @@ Currently, the following add-ons are available for Event Tickets:
 * Fix - Allow the PayPal confirmation email address sender to be empty, so it can default to the WordPress site email address [122745]
 * Fix - Stop claiming that the Attendee Registration page is an archive, add shortcode to display on any page [123044]
 * Fix - Remove CSS that was hiding the RSVP form when Blocks are disabled [123136]
+* Fix - Prevent classic rsvp form showing on blocks posts front-end [124394]
```
Comment (bordoni) [inline readme.txt]: Should we go Full caps on RSVP?

File: src/Tribe/RSVP.php [hunk 2]
```diff
@@ -1074,6 +1074,12 @@ public function front_end_tickets_form( $content ) {
 			return;
 		}
 
+		// Bail on gutenberg
+		$template_overwrite = tribe( 'tickets.editor.template.overwrite' );
+		if ( tribe_get_option( 'toggle_blocks_editor' ) && ( has_blocks( $post->ID ) || ! $template_overwrite->has_classic_editor( $post->ID ) ) ) {
```
Comment (bordoni) [inline src/Tribe/RSVP.php]: Multiple lines.

File: readme.txt (hunk 1 above)
Comment (sc0ttkclark) [reply readme.txt]:
```suggestion
* Fix - Prevent the classic RSVP form from showing in block-enabled posts on front-end [124394]
```

**Comment by sc0ttkclark** [comment]
@bordoni ready for another look

**Pull request merged**